org.eclipse.winery.topologymodeler.ui/src/app/canvas/canvas.component.ts

```diff
@@ -29,12 +29,15 @@ import {TopologyRendererActions} from '../redux/actions/topologyRenderer.actions
 export class CanvasComponent implements OnInit, OnDestroy {
   allNodeTemplates: Array<TNodeTemplate> = [];
   allRelationshipTemplates: Array<TRelationshipTemplate> = [];
+  relationshipTemplates: Array<TRelationshipTemplate> = [];
+  nodeTypes: any[] = [];
```

Comment:
Isn't it possible to assign a type here? I assume, it is not possible, because the types from the backend are not 100% fixed.

Then, please add a comment why this is `any`.

Reply:
This is no more in use, I just forgot to remove the variable.